src/replicache.ts

```diff
@@ -224,11 +224,12 @@ export default class Replicache implements ReadTransaction {
    * implements `AsyncIterable`. It also has methods to iterate over the `keys`
    * and `entries`.
    * */
-  scan({prefix = '', start, indexName}: ScanOptions = {}): ScanResult {
+  scan({prefix = '', start, limit, indexName}: ScanOptions = {}): ScanResult {
```

Comment:
Can we share the type definition with ReadTransaction to avoid this problem in the future?

Reply:
Not sure what problem you are referring to?

`limit` was never implemented. The issue was lack of tests.

Reply:
Nevermind, misread change.